### PR TITLE
Product page

### PR DESCRIPTION
**Pull request by dev-prabhat**

### CODE REVIEW

**Review by DivyanshSareen — APPROVED** [review]
Looks good. Good to merge

**Review by Muzammil-khan710 — APPROVED** [review]
Good to merge. !

File: pages/productPage/productPage.css [hunk 1]
```diff
@@ -0,0 +1,32 @@
+.grid-container {
+  display: grid;
+  grid-template-rows: auto 1fr;
+  grid-template-areas:
+    "nav-bar nav-bar nav-bar nav-bar"
+    "filter product product product";
+}
```
Comment (varun13169) [inline pages/productPage/productPage.css:7]: nit on naming

filter -> sidebar
product -> content


as tomorrow you can change the filter to something else and product can be services as well.

File: pages/productPage/productPage.css [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+
+.nav-bar {
+  grid-area: nav-bar;
+  padding: 0rem 1rem;
+  border-bottom: 1px solid var(--dark-theme-shade-20);
```
Comment (varun13169) [inline pages/productPage/productPage.css:12]: what does shade 20 means?

specify direction as well, eg: lighter or darker

**Review by varun13169 — CHANGES_REQUESTED** [review]

File: pages/productPage/productPage.css (hunk 2 above)
Comment (dev-prabhat) [reply pages/productPage/productPage.css:12]: there are 5 shades of the same color with every increasing value shades are darker

File: pages/productPage/productPage.css (hunk 1 above)
Comment (varun13169) [reply pages/productPage/productPage.css:7]: Resolved.

File: pages/productPage/productPage.css (hunk 2 above)
Comment (varun13169) [reply pages/productPage/productPage.css:12]: ok

**Review by varun13169 — APPROVED** [review]

**Pull request merged**